protocols/virtualfund/virtualfund-single-hop_test.go

```diff
@@ -321,7 +347,7 @@ func TestSingleHopVirtualFund(t *testing.T) {
 			if err != nil {
 				t.Error(err)
 			}
-			if updated.V.PreFundSignedByMe() != true {
+			if updated.V.SignedStateForTurnNum[0].HasSignatureForParticipant(bob.role) != true {
```

Comment:
Since the `testUpdate` function and `testCrank` are using the same objective and channels this had to be changed to allow for the state to be signed in `testCrank`.

We should probably be generating data per test instead of sharing it so we don't have to worry about one test influencing another.

Reply:
I can see why sharing the data is a bad pattern here... but can you explain how does this change get around the problem?